editor/project_manager.cpp

```diff
@@ -1042,10 +1042,18 @@ void ProjectListItemControl::set_project_icon(const Ref<Texture2D> &p_icon) {
 	project_icon->set_texture(p_icon);
 }
 
+bool _project_feature_looks_like_version(const String &p_feature) {
+	return p_feature.contains(".") && p_feature.substr(0, 3).is_numeric();
+}
+
 void ProjectListItemControl::set_unsupported_features(const PackedStringArray &p_features) {
 	if (p_features.size() > 0) {
-		String unsupported_features_str = String(", ").join(p_features);
-		project_unsupported_features->set_tooltip_text(TTR("The project uses features unsupported by the current build:") + "\n" + unsupported_features_str);
+		if (p_features.size() == 1 && _project_feature_looks_like_version(p_features[0])) {
+			project_unsupported_features->set_tooltip_text(TTR("This project was built in a different Godot version: ") + p_features[0]);
+		} else {
+			String unsupported_features_str = String(", ").join(p_features);
+			project_unsupported_features->set_tooltip_text(TTR("This project uses features unsupported by the current build:") + "\n" + unsupported_features_str);
+		}
```

Comment:
This makes the warning exclusive, but it's possible to catch both of them at once. We should handle this correctly and compose a tooltip with everything that is relevant based on the tags.

Reply:
That would significantly increase the code complexity and lengthen the text, is it worth doing? Also note that when double-clicking the project to edit, it will already show a more detailed message.

Reply:
> is it worth doing?

Yes. What's the reason not to show a relevant warning? Making implementing code a few lines longer is not a good reason. I would also argue that the version warning in this case less relevant than the missing features one, so it makes little sense to give it space and hide more important information.

This can also be implemented as a separate icon, so tooltip won't become too big, and the indication is immediately clear without even a hover.

Reply:
Good points, updated the PR.